celer/tests/test_lasso.py

```diff
@@ -218,6 +218,27 @@ def test_weights_lasso():
     np.testing.assert_raises(ValueError, clf1.fit, X=X, y=y)
 
 
+def test_infinite_weights():
+    np.random.seed(1)
+    n_samples, n_features = 50, 100
+    X = np.random.randn(n_samples, n_features)
```

Comment:
we have a function in utils.testing to generate the data directly (see how it's done in other tests)